pkg/acr/acr.go

```diff
@@ -64,13 +83,13 @@ func newListRepositoriesCommand(registry string) *exec.Cmd {
 	return cmd
 }
 
-func getRepositoriesFromStringData(data string) []string {
+func getRepositoriesFromStringData(data string) ([]string, error) {
 	repositories := make([]string, 0)
 	err := yaml.Unmarshal([]byte(data), &repositories)
 	if err != nil {
-		return repositories
+		return repositories, err
 	}
-	return repositories
+	return repositories, nil
```

Comment:
I think here we can just `return repositories, err` - same result.
Or `return nil, err` when `err != nil` - returned array is not used in this case.

Reply:
Fixed. Agree that `return nil, err` is more readable when err != nil, and `return repositories, nil` otherwise.